packages/flutter/test/widgets/build_owner_test.dart

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/widgets.dart';
+
+void main() {
+  testWidgets('BuildOwner.isBuilding is true inside didChangeDependencies/build and false otherwise',
+    (WidgetTester tester) async {
+      expect(tester.binding.buildOwner.isBuilding, isFalse);
+
+      await tester.pumpWidget(
+        _Stateful(
+          didChangeDependencies: () {
+            expect(tester.binding.buildOwner.isBuilding, isTrue);
+          },
+          builder: (BuildContext _) {
+            expect(tester.binding.buildOwner.isBuilding, isTrue);
+            return Container();
+          },
+        ),
+      );
+
+      expect(tester.binding.buildOwner.isBuilding, isFalse);
```

Comment:
How about `initState`?
On that timing, tester.binding.buildOwner.isBuilding should be `isTrue` 🤔

Reply:
That's expected IMO. Many built in Inheritedwidget are currently inaccessible from initState, where it is recommended to use didChangeDependencies instead